src/renderers/shared/ReactPerf.js

```diff
@@ -344,14 +372,18 @@ function getMeasurementsSummaryMap(measurements) {
 }
 
 function start() {
+  returnWarnIfDevFalse();
   ReactDebugTool.beginProfiling();
 }
 
 function stop() {
+  returnWarnIfDevFalse();
+  alreadyWarned = false;
```

Comment:
No, I mean we call it everywhere because we don’t know which method gets called first, but I want the warning to only appear once, so let’s not reset the boolean variable ever.